arrow-core/src/main/kotlin/arrow/core/extensions/nonemptylist/apply/NonEmptyListApply.kt

```diff
@@ -89,31 +88,8 @@ fun <A, B, Z> Kind<ForNonEmptyList, A>.map2Eval(
 ): Eval<Kind<ForNonEmptyList, Z>> =
   arrow.core.NonEmptyList.apply().run {
     this@map2Eval.map2Eval<A, B, Z>(arg1, arg2) as
-    arrow.core.Eval<arrow.Kind<arrow.core.ForNonEmptyList, Z>>
-}
-
-@JvmName("map")
-@Suppress(
-  "UNCHECKED_CAST",
-  "USELESS_CAST",
-  "EXTENSION_SHADOWED_BY_MEMBER",
-  "UNUSED_PARAMETER"
-)
-@Deprecated(
-  "@extension kinded projected functions are deprecated",
-  ReplaceWith(
-  "map(arg0, arg1, arg2)",
-  "arrow.core.NonEmptyList.map"
-  ),
-  DeprecationLevel.WARNING
-)
-fun <A, B, Z> map(
-  arg0: Kind<ForNonEmptyList, A>,
-  arg1: Kind<ForNonEmptyList, B>,
-  arg2: Function1<Tuple2<A, B>, Z>
-): NonEmptyList<Z> = arrow.core.NonEmptyList
-   .apply()
-   .map<A, B, Z>(arg0, arg1, arg2) as arrow.core.NonEmptyList<Z>
```

Comment:
Why is this removed?

Reply:
I guess I removed it accidentally. I will put it back.

Reply:
Fixed, I messed up a bit that part of the code.